schema/project.js

```diff
@@ -178,6 +185,18 @@ class Project extends BaseModel {
           from: 'projects.id',
           to: 'projectVersions.projectId'
         }
+      },
+      profiles: {
```

Comment:
```suggestion
      collaborators: {
```

Just to make it super clear in context what the relation actually represents. `profiles` is kind of ambiguous.

Reply:
I called it this but could not get the related queries working

Reply:
because I wrote them wrong.